src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -92,6 +94,46 @@ public void should_create_case_from_valid_exception_record() throws Exception {
         assertThat(createdCase.getData().get("email")).isEqualTo("hello@test.com");
     }
 
+    @Test
+    @Disabled
+    public void should_idempotently_create_case_from_valid_exception_record() throws Exception {
+        // given
+        CaseDetails exceptionRecord = createExceptionRecord("envelopes/new-envelope-create-case-with-evidence.json");
+
+        // when
+        // create case callback endpoint invoked first time
+        AboutToStartOrSubmitCallbackResponse callbackResponse = invokeCallbackEndpoint(exceptionRecord);
+        String caseCcdId = getCaseCcdId(callbackResponse);
+
+        // then
+        CaseDetails createdCase = ccdApi.getCase(caseCcdId, exceptionRecord.getJurisdiction());
+        assertThat(createdCase.getCaseTypeId()).isEqualTo(BULK_SCANNED_CASE_TYPE);
+        assertThat(createdCase.getData().get("firstName")).isEqualTo("value1");
+        assertThat(createdCase.getData().get("lastName")).isEqualTo("value2");
+        assertThat(createdCase.getData().get("email")).isEqualTo("hello@test.com");
+
+        assertThat(createdCase.getData().get("bulkScanCaseReference")).isNotNull();
+        String bulkScanCaseReference = (String)createdCase.getData().get("bulkScanCaseReference");
+
+        List<Long> caseIds = ccdApi.getCaseRefsByBulkScanCaseReference(bulkScanCaseReference, "bulkscan");
+        assertThat(caseIds.size()).isEqualTo(1);
+        assertThat(caseIds.get(0)).isEqualTo(createdCase.getId());
+
+        Thread.sleep(2000L);
```

Comment:
is sleeping needed?

Reply:
Updated - used await... instead. This is how it is described in the task - wait for 2 sec until ES is refreshed